graylog2-server/src/main/java/org/graylog2/rest/resources/system/contentpacks/ContentPackResource.java

```diff
@@ -221,6 +221,10 @@ public void deleteContentPack(
             @ApiParam(name = "contentPackId", value = "Content Pack ID", required = true)
             @PathParam("contentPackId") final ModelId contentPackId) {
         checkPermission(RestPermissions.CONTENT_PACK_DELETE, contentPackId.toString());
+        if (contentPackInstallationPersistenceService.findByContentPackId(contentPackId).size() > 0) {
```

Comment:
Small suggestion: Use `.isEmpty()` instead of size.

```suggestion
        if (!contentPackInstallationPersistenceService.findByContentPackId(contentPackId).isEmpty()) {
```